### DISCUSSION TITLE
Ability to include external third-party libraries viewer in explorer

### DISCUSSION THREAD

**Discussion by scruel** [microsoft/pylance-release#5585] [Enhancement]
Just like Pycharm/IDEA:
![2024-03-05_18-05-02](https://github.com/microsoft/pylance-release/assets/16933298/ffe3f458-196d-4a72-9404-c95a8be32ea2)

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
